include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -773,341 +1629,504 @@ struct SPhysicalDeviceFeatures
+    // VK_EXT_extension_512
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_513
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_514
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_515
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_516
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_517
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_MESA_extension_518
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QCOM_image_processing2
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QCOM_filter_cubic_weights
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QCOM_ycbcr_degamma
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QCOM_filter_cubic_clamp
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_523
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_524
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_attachment_feedback_loop_dynamic_state
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_526
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_527
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_528
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_529
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QNX_external_memory_screen_buffer
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_MSFT_layered_driver
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_532
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_533
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_534
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_535
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QCOM_extension_536
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_537
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_538
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_539
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_540
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_541
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_542
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_extension_543
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_544
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_545
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_extension_546
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_NV_descriptor_pool_overallocation
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_QCOM_extension_548
+
+
     // Eternal TODO: how many new extensions since we last looked? (We're up to ext number 548)
     
     /* Nabla */
```

Comment:
I also want the `if`s in `isSubsetOf` reordered in the order the members appear here, put spaces between separate extension features/bools, keep same extension together (the formatting thats there right now)